gm4_everstone/data/gm4_everstone/loot_tables/everstone.json

```diff
@@ -17,12 +17,16 @@
                       },
                       {
                           "function": "set_name",
-                          "name": [
-                              {
-                                  "text": "Everstone",
-                                  "italic": false
-                              }
-                          ]
+                          "name": {
+							"translate": "%1$s%3427655$s",
+							"with": [
```

Comment:
Mixed tabs and spaces. I know we aren't always consistent, but at least consistency on a file base would be nice.